File: backend-app/src/main/java/tom/projects/monkeylog/repository/exercise/ExerciseRepository.java
package tom.projects.monkeylog.repository.exercise;

import org.springframework.data.jpa.repository.JpaRepository;
import tom.projects.monkeylog.model.exercise.Exercise;

import java.util.List;

public interface ExerciseRepository extends JpaRepository<Exercise, Long> {
    List<Exercise> findAllByUserId(Long userId);
}

File: backend/src/main/kotlin/tom/com/workout/logbook/repository/ExerciseRepository.kt
package tom.com.workout.logbook.repository

import org.springframework.data.jpa.repository.JpaRepository
import tom.com.workout.logbook.model.exercise.Exercise
import java.util.*

interface ExerciseRepository : JpaRepository<Exercise, UUID> {
    fun findAllByUserId(userId: UUID): List<Exercise>
}
